recipes-extended/man-pages/files/onie-bisdn-upgrade.1

```diff
@@ -9,7 +9,40 @@ onie-bisdn-upgrade \- upgrade or reinstall BISDN Linux
 [\fI\,OPTION\/\fR] [\fI\,IMAGEURL\/\fR]
 .SH DESCRIPTION
 .PP
-\fIonie-bisdn-upgrade\fP will upgrade BISDN Linux.
+\fIonie-bisdn-upgrade\fP downloads and installs a new BISDN Linux image on the
+switch.
```

Comment:
`onie-bisdn-upgrade` does not download the new image by default, so I think we should not claim that.

Maybe something like

```suggestion
\fIonie-bisdn-upgrade\fP initiates installation of a new BISDN Linux image on the
switch.
```

?

Reply:
Done.